runtime/list_test.go

```diff
@@ -353,6 +353,39 @@ func TestListIteratorIter(t *testing.T) {
 	}
 }
 
+func TestListPop(t *testing.T) {
+	fun := newBuiltinFunction("TestListPop", func(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
```

Comment:
Looks like I cannot pass `l` and `args...` together to wrapArgs. 
Should I create new slice there inside the function?